src/components/Input/Input.tsx

```diff
@@ -180,7 +180,7 @@ class Input extends AutoControlledComponent<Extendable<IInputProps>, any> {
 
     return (
       <ElementType className={classes.root} {...restProps} onChange={onChange}>
-        {createHTMLInput(type, {
+        {createHTMLInput(input || type, {
```

Comment:
have we agreed to have this `input` prop for the `Input` element (like as it is in SUIR)? We've had some concerns - specifically, the fact that it makes it non-intuitive for the client what `input` prop of `Input` component is about. Maybe we should leave it for a dedicated PR?

Reply:
then we won't be able to achieve functionality of `RadioGroup` items that contain inputs.. Marija told we we're expected to merge a dedicated PR as well; I'm ok with removing it from here but the examples with `input` in this PR won't work correctly

@kuzhelov @mnajdova